src/test/java/com/jcabi/http/mock/MkAnswerMatchersTest.java

```diff
@@ -77,4 +77,51 @@ public void canMatchHeader() {
             )
         );
     }
+
+    /**
+     * MkAnswerMatchers can match MkAnswer binary body.
+     */
+    @Test
+    public void canMatchBinaryBody() {
+        final byte[] body = this.getAllByteValues();
+        final MkAnswer query = Mockito.mock(MkAnswer.class);
+        Mockito.doReturn(body).when(query).content();
+        MatcherAssert.assertThat(
+            query,
+            MkAnswerMatchers.hasContent(
+                Matchers.is(body)
+            )
+        );
+    }
+
+    /**
+     * MkAnswerMatchers can show mismatch of MkAnswer binary body.
+     */
+    @Test
+    public void canNotMatchBinaryBody() {
+        final byte[] body = new byte[]{0};
+        final MkAnswer query = Mockito.mock(MkAnswer.class);
+        Mockito.doReturn(body).when(query).content();
```

Comment:
@westonized this is not specified in our code style, I was just curious because in other projects I've seen different syntax